packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -42,19 +42,33 @@ class FuchsiaSdk {
   /// Returns the fuchsia system logs for an attached device.
   ///
   /// Does not currently support multiple attached devices.
-  Stream<String> syslogs() {
+  Stream<String> syslogs(String id) {
```

Comment:
cc @chingjun, this change should be safe to roll but we'll need to tweak it in google3 too

Reply:
Does this change mean that the comment right above this line can be removed now?